.github/workflows/codeql-analysis.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      uses: github/codeql-action/autobuild@v2
```

Comment:
Fixes CI warning (same for below)

Reply:
Moved to separate PR: https://github.com/vmware-tanzu/function-buildpacks-for-knative/pull/138